.gitignore

```diff
@@ -1,2 +1,7 @@
 node_modules
 reset.css
+icons/*.tsx
```

Comment:
why are the icons not in this repo? Because of the icon license?

Reply:
The SVG icons _are_ in the repo, the generated files are not.

Reply:
Sorry, not precise language. The SVGs are under source control, the compiled files are not (they are generated on the fly, we don't care what their history looks like)